### PR TITLE
Update with Config auto

### PR DESCRIPTION
**Pull request by s-santillan**
(Draft) Updated with information about config auto.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ Welcome! This repository is the “standard library” for [Semgrep](https://sem
 
 ## How do I use these rules?
 
-We recommend starting with a pre-selected subset of rules that someone has already carefully tuned for precision. These subsets of rules, which we will call rulesets, may include rules not present in this repository. However, they are available for free on the [Semgrep Registry](https://semgrep.dev/explore).
+We recommend running Semgrep with the command `semgrep --config auto`. The `--config auto` flag will automatically fetch rules relevant to your project from [Semgrep Registry](https://semgrep.dev/explore). Only the name of your project will be sent to Semgrep Registry, to look up the project's language and select relevant rules. Your source code will not be uploaded.
+
+Alternatively, you can start with a pre-selected subset of rules that has already been carefully tuned for precision. These subsets of rules, which we call rulesets, may include rules not present in this repository. However, they are available for free on the [Semgrep Registry](https://semgrep.dev/explore). These rules have been written by open-source contributors and r2c.
```
Comment (s-santillan) [inline README.md:17]: In the previous messaging, the rules were written by "someone" — who are these people? I assumed "open-source contributors" and r2c (last part of the sentence.) Please let me know if this is correct?

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ Welcome! This repository is the “standard library” for [Semgrep](https://sem
 
 ## How do I use these rules?
 
-We recommend starting with a pre-selected subset of rules that someone has already carefully tuned for precision. These subsets of rules, which we will call rulesets, may include rules not present in this repository. However, they are available for free on the [Semgrep Registry](https://semgrep.dev/explore).
+We recommend running Semgrep with the command `semgrep --config auto`. The `--config auto` flag will automatically fetch rules relevant to your project from [Semgrep Registry](https://semgrep.dev/explore). Only the name of your project will be sent to Semgrep Registry, to look up the project's language and select relevant rules. Your source code will not be uploaded.
```
Comment (s-santillan) [inline README.md]: I specified that 
1. their project name is sent to Semgrep Registry (is this actually where it's sent, or should I say something more generic?)
2. The only reason why is to look up the project's language makeup and select relevant rules. Please let me know if this is correct :)

Comment (ievans) [inline README.md]:
```suggestion
We recommend starting with `semgrep --config auto`. The `--config auto` flag will fetch rules relevant to your project from [Semgrep Registry](https://semgrep.dev/explore). The name of your project will be sent to Semgrep Registry as an identifier to make selecting relevant rules fast next time; source code will not be uploaded.
```

File: README.md (hunk 1 above)
Comment (ievans) [reply README.md:17]: Yes!

**Review by ievans — APPROVED** [review]
:)

**Pull request merged**